src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        <UserInfo post={post} />
```

Comment:
It seems that the 'userInfo' component is expecting a 'User' prop but a 'post' prop is being passed instead. Make sure to pass the correct prop.